### DISCUSSION TITLE
Mini switches not switching but making led's flashing

### DISCUSSION THREAD

**Discussion by videobuff** [arendst/Tasmota#18622] [Support request]
I have two mini switches (One SonOff and one Shelly 1) flashed with the latst Tasmota (12.5.0) and both have my garden LED's flashing when in the OFF state. The Sonoff Mini does make light go on, but the Shelly also makes the leds flash when in the ON state. Total wire length in the garden is 10 meters and 5 leds are connected in 2 meter intervals.

I am at a loss as where to look for a solution ?

**Comment by barbudor** [comment]
I'm confused when your are talking about LEDs
What are you exactly talking about ?
Sheely 1 does have a 12VDC mode but AFAIK, not the Sonoff Mini
Can you clarify your use-case/wiring ?

**Comment by videobuff** [comment]
Helo Barbydur.
The switches are mounted in a waterproof case against the house wall, and from there  I run a 10-15 meter long ground cable through the garden. At intervals of 5 meter I connect a 220V LED garden lamp (3 watt)
When I switch the garden lights ON, the flash in stead of continuously on. But they also flash when the switch is in the OFF state.
On my desk, everything worked normal. 
Could it be too much capacitance of the 220V wires or induction or even a low resistance to the ground ? 
I am going to test more tomorrow since it is raining right now.

Erik Schott
***@***.***

Mijn 2e email adres ***@***.*** vervalt per 01-08-2023

**Reply by barbudor** [reply]
Just to rule out the wiring, may be you can make a test with just a single lamp on a short cable

Does the flashing issue occurs when you click on the toggle button on the Web UI ?

I see 2 possibilities for "flashing"
- a command loop
- the Blink feature

1) a command loop
If you have any kind of rule or other external process that react to a power state change by sending a "Power Toggle" command, then it may create a loop when clicking on the web GUI to set Power ON, would trigger the "rule" which would toggle to OFF, which would trigger the rule, which would toggle to ON etc...

It could happen if you f..k-up the "Fulltopic" forgetting to include a "%prefix%" or hardcoding "cmnd" in the FullTopic

I suggest to look at the console logs to see what happens when the LEDs are flashing.
Does anything happens ?
Try "Weblog 3" to have more details

2) the blink feature
There's one feature in tasmota that could make the relay output to "flash": It's the blink mode
It is started with a command "Power 3" or "Power Blink".
The number of consecutive "blink" is set by the command "BlinkCount" with value 0 meaning infinite and the period of blinks is set with "BlinkTime"
Note that blinking cannot be started by clicking on the toggle button on the Web UI, so it it's flashing when using the Web UI, Blink is not the cause of your problem.
